src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import { getColor, pxToRem } from '../../utils/helpers';
+import { LabelProps, TabProps } from './Tabs.types';
+import { ColorTypes } from '../../theme/colors.enums';
+
+const Label = styled.a<LabelProps>`
+  font-weight: 500;
+  font-size: ${pxToRem(18)};
+  line-height: ${pxToRem(20)};
+  padding-bottom: ${pxToRem(3)};
+  cursor: pointer;
+  border-bottom: 2px solid
+    ${({ $isSelected, color }) =>
+      $isSelected ? getColor(color) : getColor('graphiteHB')};
+
+  &:hover {
+    border-bottom: 2px solid ${({ color }) => color};
+  }
+  &:not(:last-of-type) {
+    margin-right: ${pxToRem(30)};
+  }
+`;
+
+const Tab: React.FC<TabProps> = ({
+  children,
+  isSelected,
+  onClick,
+  color = ColorTypes.blueberryClassic,
+  value,
+}) => {
+  const isLink = value?.toString()?.startsWith('/');
+  const handler = isLink ? { to: value } : { onClick: () => onClick(value) };
+
+  return (
+    <Label
+      $isSelected={isSelected}
```

Comment:
why prefix with `$` sign?

Reply:
transient props, they will not be passed to rendered DOM so no more errors 🎉

Reply:
https://styled-components.com/docs/api#transient-props

Reply:
cool, didn't know that :)